css/styles.css

```diff
@@ -386,3 +388,100 @@ button.form_btn:hover {
   grid-template-rows: 25% 25% 25% 25%;
   row-gap: 20px;
 }
+
+.contenido-form {
+  display: flex;
+}
+
+/*      Desktop Style     */
+
+@media (min-width: 768px) {
+  .portfolio {
+    display: flex;
+    display: none;
+  }
+
+  .portfolio1 {
+    display: flex;
+    width: auto;
+  }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .card1 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .project-detail {
+    width: 569px;
+  }
+
+  .contenido1 {
+    display: flex;
+    width: 50%;
+    height: auto;
+    flex-direction: column;
+  }
+
+  .contenido-mobil {
+    display: none;
+  }
+
+  .contenido-desktop {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    width: auto;
+  }
+
+  .contenido-lenguajes {
+    display: flex;
+  }
+
+  .form {
+    display: flex;
+    background-image: url(../img/contact-background-big.png);
```

Comment:
- [ ] Please correct the exhibition of this background image. It's a project requirement.